checker.go

```diff
@@ -0,0 +1,166 @@
+package wraperr
+
+import (
+	"go/ast"
+	"go/token"
+	"go/types"
+	"strings"
+)
+
+type Checker interface {
+	Check(ReportFunc)
+}
+
+type ReportFunc func(assignedAt, returnedAt token.Pos)
+
+func NewChecker(
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewChecker should have comment or be unexported